README.md

```diff
@@ -894,6 +894,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [The Odin Project](https://www.theodinproject.com/) - Free, open-source platform with a curriculum focused on JavaScript and Ruby for web development.
   * [Free ATS Resume AI Solutions](https://ayehigh.com) - Free platform with lots of GenAI Powered Resume Solutions helpful for Analyzing, Rewriting, Grooming and Shortlisting Resumes and help in Career Growth.
   * [Free Professional Resume Templates & Edtior](https://www.overleaf.com/latex/templates/tagged/cv) - Free platform with lots of Resume templates of Experienced Professionals, ready to clone and edit fully and download, ATS optimized.
+  * [CodeCrafters](https://codecrafters.io/) - Get really good at your craft by building dev tools from scratch (Build your own Redis, Git, Docker, SQLite)   
```

Comment:
<img width="345" alt="Screenshot 2024-04-22 at 12 34 14" src="https://github.com/ripienaar/free-for-dev/assets/82342/9bad3a1f-c130-4929-9555-5bb5413fbc75">

Does this mean you cant finish a project on the free plan because it has more than 2 stages?

Reply:
Good catch! I’ve updated the tooltip. We actually have 2 entirely free challenges at the moment (Redis and HTTP Server), this changes from month to month.

![image](https://github.com/ripienaar/free-for-dev/assets/3888224/0a00dbd4-0c74-454f-80ef-b2330b73441f)

Reply:
OK, please update the description here saying 2 projects can be done in the free tier.  But I still dont find that description very clear, maybe a FAQ entry describing that you have always 2 free projects would clarify